src/Api/Model/Shared/Rights/SiteRoles.php

```diff
@@ -8,7 +8,7 @@
 class SiteRoles extends RolesBase
 {
     const SITE_MANAGER = "site_manager"; // highest possible level for site: various management responsibilities like creating projects
-    const PROJECT_CREATOR = "project_creator"; // permission to create a project
+    const PROJECT_OWNER = "project_owner"; // permission to create a project, delete a project, and transfer ownership of that project
```

Comment:
`project_creator` is a string stored in Mongo. I would guess that changing it here will break things.
Renaming the variable sounds like a good idea though. 👍

![image](https://user-images.githubusercontent.com/12587509/202477509-5f01a1fc-e6fe-47c3-a562-d362a1c8e184.png)

Reply:
Thanks, great point. I changed it back. For me, though, I'm still seeing some potential issues with the permissions in the case that you are showing (an empty project). Working on that...

Reply:
Okay, from testing on production (languageforge.org), I see that we have some quirks in how the editor component appears when an empty project is shared with an editor or commentor (but not a manager); the quirks that I'm seeing are happening on production and aren't a result of the PR, so I'll make a separate issue.